var/spack/repos/builtin/packages/ipopt/package.py

```diff
@@ -144,6 +156,11 @@ def configure_args(self):
                     ]
                 )
 
+        if spec.satisfies("+java"):
+            args.extend(["--enable-java"])
+        else:
+            args.extend(["--disable-java"])
+
```

Comment:
You could simplify this using the `enable_or_disable` helper (see docs at https://spack.readthedocs.io/en/latest/build_systems/autotoolspackage.html#addings-flags-to-configure).

Reply:
@chapman39: Tamara's suggestion is good, try it with the next update, but this should be fine.

Reply:
good to know, thanks!